src/components/queue/job/JobFiltersBar.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div class="flex items-center justify-between gap-2 px-3">
+    <div class="min-w-0 flex-1 overflow-x-auto">
+      <div class="inline-flex items-center gap-1 whitespace-nowrap">
+        <button
+          v-for="tab in jobTabs"
+          :key="tab"
+          class="h-6 rounded border-0 px-3 py-1 text-[12px] leading-none hover:opacity-90"
+          :class="[
+            selectedJobTab === tab
+              ? 'bg-[var(--color-charcoal-500)] text-white'
+              : 'bg-transparent text-[var(--color-slate-100)]'
```

Comment:
Nit:
But also, should these use design tokens so they'll adapt to light/dark mode?
```suggestion
            selectedJobTab === tab
              ? 'bg-charcoal-500 text-white'
              : 'bg-transparent text-slate-100'
```

Reply:
yeah I see how the entire component is missing this, just a sec